pages/signup/Signup.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { FaChevronLeft } from 'react-icons/fa';
+import styles from './signup.module.scss';
+
+export default function Signup() {
+	const Back = () => (<button className={styles.back}><FaChevronLeft /> Back </button>);
```

Comment:
If you create a variable that returns jsx please have it be its own component, this button could potentially be reused in other places at some point.

Reply:
✔️